pstress/pstress-run.sh

```diff
@@ -859,11 +859,11 @@ EOF
     if [[ ${TRIAL} -gt 1 && $REINIT_DATADIR -eq 0 ]]; then
       mkdir -p ${RUNDIR}/${TRIAL}/
       echoit "Copying datadir from $WORKDIR/$((${TRIAL}-1))/node1 into ${RUNDIR}/${TRIAL}/node1 ..."
-      rsync -ar --exclude='*core*' ${WORKDIR}/$((${TRIAL}-1))/node1/ ${RUNDIR}/${TRIAL}/node1/ 2>&1
```

Comment:
What is this fix for? IS it for PXC

Reply:
This fix is unrelated to the previous question asked by you. This change just creates the log in the trial directory instead of trial/node1. Earlier log files were created inside node1.
Reason for this change is just for symmetry since we are creating the logs at the same place for PS runs also